### PR TITLE
599 adminlock should scale

### PR DESCRIPTION
**Pull request by AlexeyKosinski**
This PR closes https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/599
Try run batch request on local and fuse blockchain but transaction returns 
```
 { err: null, res: Result {} }
```

* documentation https://web3js.readthedocs.io/en/v1.2.1/web3.html#batchrequest

local
image

fuse
image

### CODE REVIEW

File: batch.js [hunk 1]
```diff
@@ -85,21 +85,30 @@ const getContract = (id) => {
...
 const test = async () => {
-  setProvider('http://localhost:9545')
-  // getFirstWallet('drip industry pizza deny pistol stem must device citizen crowd offer now') // fuse
-  // contract = getContract(121) //fuse
-  getFirstWallet('myth like bonus scare over problem client lizard pioneer submit female collect') // local
-  contract = getContract(4447) //local
+  // FUSE
+  // setProvider('wss://explorer-node.fuse.io/ws')
+  setProvider('https://rpc.fuse.io/')
+  getFirstWallet('drip industry pizza deny pistol stem must device citizen crowd offer now') // fuse
```
Comment (sirpy) [inline batch.js]: exposing secrets!

Comment (AlexeyKosinski) [reply batch.js]: trying version web3@1.0.0-beta.55
results:
image

Comment (AlexeyKosinski) [reply batch.js]: remove mnemnic.

test your commit with local blockchain - works

Comment (AlexeyKosinski) [reply batch.js]: but it is not working in fuse 
if websocket provider - it is just stuck 
if http provider - have this error:

image

Comment (AlexeyKosinski) [reply batch.js]: it works with fuse if we add tx to batch like this
```
batch.add(
    contract.methods
      .whiteListUser('0x919b99dcabae3fea4909af23b8dfa9f2d2c273de', 'ab')
      .send({...params,nonce}, (err, res) => console.log('TX 3: ', { err, res }))
  )```
not like this
```
batch.add(
    contract.methods
      .whiteListUser('0x919b99dcabae3fea4909af23b8dfa9f2d2c273de', 'ab')
      .send.request({...params,nonce}, (err, res) => console.log('TX 3: ', { err, res }))
  )
```

**Pull request closed without merging**